python/ray/node.py

```diff
@@ -143,7 +143,7 @@ def __init__(
             ),
             setup_worker_path=os.path.join(
                 os.path.dirname(os.path.abspath(__file__)),
-                f"workers/{ray_constants.SETUP_WORKER_FILENAME}",
+                "workers", f"{ray_constants.SETUP_WORKER_FILENAME}",
```

Comment:
This fixes the `/` for windows by using `os.path.join` with multiple arguments. The previous code worked, but looks strange in debug logs